src/Filter/BooleanFilter.php

```diff
@@ -24,9 +24,7 @@
 class BooleanFilter extends Filter
 {
     /**
-     * @param string $alias
-     * @param string $field
-     * @param mixed  $value
+     * NEXT_MAJOR: Remove $alias parameter.
```

Comment:
FYI: The signature is defined in the parent class
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Filter/FilterInterface.php#L34-L36